jina/drivers/querylang/filter.py

```diff
@@ -37,10 +37,12 @@ def __init__(self, lookups: Dict[str, Any], *args, **kwargs):
         self._lookups = Q(**lookups) if lookups else None
```

Comment:
```suggestion
        self._lookups = lookups
```

I think `Q` builder can be delayed to `__apply_all`